tracer/test/Datadog.Trace.Debugger.IntegrationTests/Approvals/snapshots/ProbeTests.GreaterThanArgumentTrueAtEntry.verified.txt

```diff
@@ -47,10 +43,6 @@
               }
             },
             "staticFields": {
-              "Dsl": {
-                "type": "String",
-                "value": "{\r\n  \"dsl\": \"ref intArg > 2\"\r\n}"
-              },
```

Comment:
Why all of these got removed? 🤔

Reply:
It added a lot of verbosity to approval files and tests classes even though we didn't really consider them in any way - we always referred to json only.